### PR TITLE
Make sources.list record abstract for Debian

### PR DESCRIPTION
**Pull request by Himura2la**
# PR Summary

Remove duplicate texts by using system variables in shell snippets

## PR Context

Installation Instructions

Version(s) of documents impacted

- [x] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes reference to related issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

**Comment by Himura2la** [comment]
Nice validation!

**Comment by sdwheeler** [comment]
@TravisEz13 Can you review this change?

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 1]
```diff
@@ -49,67 +49,17 @@ various package managers:
...
-### Installation via Package Repository - Ubuntu 18.04
-
-PowerShell Core for Linux is published to package repositories for easy installation and updates.
-
-The preferred method is as follows:
-
-```sh
-# Download the Microsoft repository GPG keys
-wget -q https://packages.microsoft.com/config/ubuntu/18.04/packages-microsoft-prod.deb
+wget -q https://packages.microsoft.com/config/ubuntu/$(. /etc/os-release && echo $VERSION_ID)/packages-microsoft-prod.deb
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: Please revert this type of change.  Unfortunately these need to stay hard coded for synchronization with other teams.

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 2]
```diff
@@ -130,77 +80,37 @@ pwsh
...
-Download the Debian package `powershell_6.2.0-1.ubuntu.18.04_amd64.deb` from the [releases][] page
-onto the Ubuntu machine.
+### Installation via Direct Download - Ubuntu 16.04 and 18.04
 
-Then, in the terminal, execute the following commands:
+Execute the following commands in the terminal:
 
 ```sh
-sudo dpkg -i powershell_6.2.0-1.ubuntu.18.04_amd64.deb
+wget -qO pwsh.deb https://github.com/PowerShell/PowerShell/releases/download/v6.2.3/powershell_6.2.3-1.ubuntu.$(. /etc/os-release && echo $VERSION_ID)_amd64.deb
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: again revert

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md [hunk 3]
```diff
@@ -215,7 +125,7 @@ sudo apt-get install -y curl gnupg apt-transport-https
 curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
 
 # Register the Microsoft Product feed
-sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-debian-stretch-prod stretch main" > /etc/apt/sources.list.d/microsoft.list'
+sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-$(. /etc/os-release && echo $ID-$VERSION_CODENAME-prod $VERSION_CODENAME) main" > /etc/apt/sources.list.d/microsoft.list'
```
Comment (TravisEz13) [inline reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: again revert

File: reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md (hunk 1 above)
Comment (TravisEz13) [reply reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md]: Basically don't try to guess which one is correct.

**Comment by TravisEz13** [comment]
Uninstall changes seem okay.

**Comment by Himura2la** [comment]
I'm afraid there's no way to leave uninstall changes only, because they depend on a new structure with minor versions merged. If you can't accept variable sources, then, I guess, the PR can be closed.

**Comment by sdwheeler** [comment]
Closing.

**Comment by TravisEz13** [comment]
@Himura2la We have to coordinate with partners, who don't allow variable sources.  It should work the way you suggested.

**Comment by Himura2la** [comment]
I tested the code today and found that `$VERSION_CODENAME` is absent in debian 9's `/etc/os-release` file. Please note this if you continue the work on this

**Pull request closed without merging**